Profile byline and page 2 stuff

mipli opened the pull request:
This PR adds support for vertical profile bylines
![1588930388](https://user-images.githubusercontent.com/795482/81392936-bbdc1d80-911f-11ea-846e-31d27cad113d.png)

and the "lest siden sist" column the DN has on page 2 on Saturdays. 
![1588930514](https://user-images.githubusercontent.com/795482/81393089-ffcf2280-911f-11ea-979e-e7e103b6ed53.png)

mipli committed:
Vertical layout, step 1

mipli committed:
Align around separator lines

mipli committed:
Byline box sizing

mipli committed:
Byline top aligned over separator

mipli committed:
Start of lest siden sist

mipli committed:
Added some colours

mipli committed:
More colour stuff

mipli committed:
padding work

mipli committed:
Added more margins

mipli committed:
Update after InDesign lookup

mipli committed:
Email border and other style updates

mipli committed:
Lest siden sist bullet point update

rudolfrck commented:
Looks good to me. Seems to be a mix between spaces and tabs here. Could you convert it all to tabs? And I think I'll squash it when I merge.

mipli committed:
Update to use tabs instead of spaces

mipli commented:
@rudolfrck I've updated all the files touched to use tabs now, and added an `.editorconfig` file to ensure that we keep the same coding style in the future as well.

peterudo commented on templates/components/text-column/text-column.scss:
```diff
@@ -419,3 +419,50 @@
+		left: -4.85mm;
+		color: var(--color-base-dark);
+		@media print {
+			color: var(--color-base-dark-cmyk);
```
Could you instead of doing this, add the `--color-base-dark` to the `@media print` definition in `themes/dn/_styles.scss`?

mipli replied:
Changed that now

mipli replied:
But looking a bit more at the styling in `_styles.scss` I'm not sure this will work, since we in the various theme sections overwrite both `--color-base-dark` and `--color-base-dark-cmyk` with new values taken from the SCSS variables.

Should probably rewrite all of these components to not use `...-cmyk` colors anywhere, and only use those in a `@media print` block at root level. But I'm not sure this PR is the right place for that.

peterudo replied:
Yes, the theme stuff should be updated too.

mipli replied:
I'm not sure how the other theme stuff is used, and I don't have examples of those templates locally to test the changes. Is it okay if we save that for another PR?

mipli committed:
Set base dark cmyk value

mipli committed:
Revert "Set base dark cmyk value"

This reverts commit 30025d23cb574e900dc81b03b786d0627c8f32d2.

espenmoe commented:
Are we ready to test?

tir. 19. mai 2020 kl. 12:50 skrev Michael <notifications@github.com>:

> Merged #2 <https://github.com/aptoma/lp-assets-print/pull/2> into
> dn-redesign.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/aptoma/lp-assets-print/pull/2#event-3351250601>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AC35HQ3YUOIETUAWP2TCG3DRSJQBBANCNFSM4M4BCIXA>
> .
>


-- 
*Espen Moe*
____________________

Mobile:             +47 41 64 39 00
Twitter:                @espenmoe
Skype:                 espenmoe

mipli commented:
@espenmoe Yes, this is ready for testing now. I'll create two test templates for you to look at to see how you can use these two new features and send you and email with information about it.

espenmoe commented:
thx

tir. 19. mai 2020 kl. 13:12 skrev Michael <notifications@github.com>:

> @espenmoe <https://github.com/espenmoe> Yes, this is ready for testing
> now. I'll create two test templates for you to look at to see how you can
> use these two new features and send you and email with information about it.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/aptoma/lp-assets-print/pull/2#issuecomment-630751807>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AC35HQ6DA3D7XSGXJZABQ4DRSJSTNANCNFSM4M4BCIXA>
> .
>


-- 
*Espen Moe*
____________________

Mobile:             +47 41 64 39 00
Twitter:                @espenmoe
Skype:                 espenmoe
